components/divider/stories/divider.stories.js

```diff
@@ -45,5 +47,107 @@ export default {
 	},
 };
 
+/**
+ * The default size for divider is medium.
+ */
 export const Default = Template.bind({});
 Default.args = {};
+Default.parameters = {
+	docs: {
+		story: {
+			height: "auto",
+		}
+	},
+};
+
+const AllDividerSizes = (args) => html`
+	<div>
+		<div style="display: flex; flex-direction: column; padding: 16px;">
+			<h3 style="margin: 0;">Small</h4>
+			${Template({
+				...args,
+				size: "s"
+			})}
+		</div>
+		<div style="display: flex; flex-direction: column; padding: 16px;">
+			<h3 style="margin: 0;">Medium (default)</h4>
+			${Template({
+				...args,
+				size: "m"
+			})}
+		</div>
+		<div style="display: flex; flex-direction: column; padding: 16px;">
+			<h3 style="margin: 0;">Large</h4>
+			${Template({
+				...args,
+				size: "l"
+			})}
+		</div>	
+	</div>
+`;
+
+/*
+ * Stories for the MDX "Docs" only.
+ * "storyName" refers to the display name/heading for a component
+ */
+export const Large = Template.bind({});
+Large.tags = ["docs-only"];
+Large.args = {
+	size: "l",
+};
+Large.parameters = {
+	chromatic: { disableSnapshot: true },
+	docs: {
+		story: {
+			height: "auto",
+		}
+	},
+};
+
+export const Small = Template.bind({});
+Small.tags = ["docs-only"];
+Small.args = {
+	size: "s",
+};
+Small.parameters = {
+	chromatic: { disableSnapshot: true },
+	docs: {
+		story: {
+			height: "auto",
+		}
+	},
+};
+
+/**
```

Comment:
Small, medium, and large could be displayed within the same story, like they are for static black and static white. I don't think we need to separate them out.

Reply:
I took care of the staticWhite color and then this sizing stuff all in one commit: https://github.com/adobe/spectrum-css/pull/2833/commits/b0028c9aebbf080502de7e3e78b84880083d2325